lib/core/include/thread_pool.hpp

```diff
@@ -5,13 +5,16 @@
 
 #include <boost/asio.hpp>
 
+#include "irods_exception.hpp"
+
 namespace irods
 {
     class thread_pool
     {
     public:
         explicit thread_pool(int _size)
-            : pool_{static_cast<std::size_t>(_size)}
+            : stopped_{false}
```

Comment:
why is a stop flag needed? the thread pool does not execute anything after it has been stopped. it ignores future requests.

Reply:
it is no longer needed, cruft from a previous attempt.